src/Dashboard/Grid.php

```diff
@@ -424,7 +424,6 @@ public function show(bool $mini = false)
             rand:        '{$rand}',
             embed:       {$embed_str},
             ajax_cards:  {$ajax_cards},
-            all_cards:   {$cards_json},
```

Comment:
`$cards_json` is not used now, so it can be removed, and `$cards = $this->getAllDasboardCards();` can be removed too.